src/content_sao.h

```diff
@@ -42,9 +41,76 @@ class UnitSAO: public ServerActiveObject
 	s16 getHP() const { return m_hp; }
 	// Use a function, if isDead can be defined by other conditions
 	bool isDead() const { return m_hp == 0; }
+
+	void setArmorGroups(const ItemGroupList &armor_groups)
+	{
+		m_armor_groups = armor_groups;
+		m_armor_groups_sent = false;
+	}
+	const ItemGroupList &getArmorGroups() const { return m_armor_groups; }
+	ObjectProperties* accessObjectProperties() { return &m_prop; }
+	void notifyObjectPropertiesModified() { m_properties_sent = false; }
+	void setAnimation(v2f frame_range, float frame_speed, float frame_blend, bool frame_loop);
+	void getAnimation(v2f *frame_range, float *frame_speed, float *frame_blend, bool *frame_loop) const;
+	void setBonePosition(const std::string &bone, v3f position, v3f rotation)
```

Comment:
I'll change it, but I doubt copying 4 fewer bytes (on a 64-bit platform) will have any measurable performance impact.